Pspcl/Pspcl.Web/Controllers/AddStockController.cs

```diff
@@ -1,12 +1,25 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using Pspcl.DBConnect;
+using Pspcl.Services;
+using Pspcl.Web.Models;
 
 namespace Pspcl.Web.Controllers
 {
-    public class AddStockController : Controller
+    
+    public class AddStockController: Controller
```

Comment:
Check naming convention.. It should not be AddStockController

Reply:
changed it to stockView controller